spec/certs/store_spec.rb

```diff
@@ -11,6 +11,10 @@
     )
 
     CertificateStore.instance.reset
+    CertificateStore.instance.clear_root_identifiers
```

Comment:
yesssss test pollution :[ .... wouldn't we expect `#reset` to do all these things?

Reply:
I think clearing root identifiers make sense. Idk about busting the caches though.

Reply:
Maybe we move these to a global before action?

Reply:
Except reseting the CertStore since rebuilding it is slow.